orc8r/cloud/Makefile

```diff
@@ -108,7 +109,7 @@ fmt: $(FMT_LIST)
 $(FMT_LIST): %_fmt:
 	make -C $*/cloud/go fmt
 
-fullgen: clean_gen gen swagger tidy
+fullgen: clean_gen gen swagger nms_fullgen tidy
 
 gen: tools gen_protos $(GEN_LIST)
```

Comment:
Also add nms_gen to prereqs for gen

Reply:
Oh hm won't do that just yet - it only depends on the swagger outputs for now.

Reply:
Oh true. Okay cool is fine then, lgtm